templates/zerver/integrations/discourse.md

```diff
@@ -1,16 +1,27 @@
+# Zulip Discourse integration
+
+Get Discourse notifications in Zulip!
+
+{start_tabs}
+
 1.  {!create-an-incoming-webhook.md!}
 
-    The API keys for "Incoming webhook" bots are limited to only
-    sending messages via webhooks. Thus, this bot type lessens
-    the security risks associated with exposing the bot's API
-    key to third-party services.
+    !!! warn ""
+
+        **Note:** The API keys for "Incoming webhook" bots are limited to
+        only sending messages via webhooks. Thus, this bot type lessens
+        the security risks associated with exposing the bot's API key to
+        third-party services.
 
 1.  Copy the `USERNAME` and `API KEY` - you'll need it later.
 
-1.  Head over to the
-    [discourse-chat-integration setup instructions](https://meta.discourse.org/t/68501)
-    and complete them.
+1.  Complete the
+    [discourse-chat-integration setup instructions][setup-instructions].
+
+{end_tabs}
 
 {!congrats.md!}
 
 ![Discourse chat integration](/static/images/integrations/discourse/001.png)
```

Comment:
This is another screenshot that's pretty different from our current UI. Not sure if we should drop it or keep it. I'm sure this image/doc is listed in the issue you created specifically for the screenshot updates.